sagar/data/Lab_1sagar.ipynb

```diff
@@ -0,0 +1,1314 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        points=widgets.IntSlider(min=50, max=500, step=50, value=100)
</pre><p>Looks like points slider is not brhaving as it should. Decreasing/increasing the slider should increase/decrease the points on x axis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>